p2p/transport/quic/virtuallistener.go

```diff
@@ -70,7 +70,7 @@ func (r *acceptLoopRunner) RmAcceptForVersion(v quic.VersionNumber) {
 
 	ch, ok := r.muxer[v]
 	if !ok {
-		panic("expected chan in accept muxer")
```

Comment:
Why this change? Did you hit this panic?

Reply:
I think I accidentally committed unrelated changes. Discarding this.